fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatAuthenticationConfig.java

```diff
@@ -78,25 +77,36 @@ public FiatService fiatService(FiatClientConfigurationProperties fiatConfigurati
   }
 
   @Bean
-  FilterRegistrationBean fiatFilterRegistrationBean(FiatAuthenticationFilter filter) {
-    val frb = new FilterRegistrationBean(filter);
-    frb.setOrder(Ordered.HIGHEST_PRECEDENCE + 1);
-    frb.addUrlPatterns("/*");
-    return frb;
+  FiatWebSecurityConfigurerAdapter fiatSecurityConfig(@Value("${services.fiat.enabled:false}") boolean isFiatEnabled) {
```

Comment:
wondering about this as a @Value here and then down below as a dynamic property

I suspect if we want to go the dynamic route then we need a static spring component configuration that always sets up the beans a consistent way, and the feature flag just controls whether those no-op or actually do things

Reply:
This particular change was to support being able to hit the actuated endpoints.